### PR TITLE
Improve fixtures for test scenarios

### PR DESCRIPTION
**Pull request by palango**
And add another routing test case that I used in the blogpost as well.

### CODE REVIEW

File: pathfinder/tests/fixtures/network_service.py [hunk 1]
```diff
@@ -56,6 +56,38 @@ def channel_descriptions() -> List:
...
+
+    channel_descriptions = [
+        (0, 100,  20, 0.3, 1,  50,  10, 0.3),  # capacities  90 --  60
+        (1,  40,  10, 0.2, 4, 130, 100, 0.2),  # capacities 130 --  40
+        (0,  90,  10, 0.1, 2,  10,   0, 0.1),  # capacities  80 --  10
+        (2,  50,  20, 0.2, 3,  50,  20, 0.2),  # capacities  50 --  50
+        (3, 100,  40, 0.1, 4,  80,   0, 0.1),  # capacities  60 -- 120
+        (2,  30,  10, 0.1, 5,  40,  15, 0.1),  # capacities  35 --  35
+        (5, 500, 900, 0.1, 4, 750, 950, 0.1),  # capacities 550 -- 700
+    ]
```
Comment (err508) [inline pathfinder/tests/fixtures/network_service.py]: maybe specify the significant differences between the topologies as a comment, including expected behaviour and a 'tag' what is edgy about the test case.

File: pathfinder/tests/test_mocked_integration.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def test_pfs_with_mocked_events(
     token_networks: List[TokenNetwork],  # just used for addresses
     addresses: List[Address],
     pathfinding_service_mocked_listeners: PathfindingService,
-    channel_descriptions: List
+    channel_descriptions_case_1: List
 ):
```
Comment (err508) [inline pathfinder/tests/test_mocked_integration.py]: We could potentially make this topology independent, i.e. together with a given topology, hand over the expected results that are to be asserted by the test.

File: pathfinder/tests/fixtures/network_service.py (hunk 1 above)
Comment (palango) [reply pathfinder/tests/fixtures/network_service.py]: Done.

File: pathfinder/tests/test_mocked_integration.py (hunk 2 above)
Comment (palango) [reply pathfinder/tests/test_mocked_integration.py]: I tried this, but right now the tests seem to different for this to make sense. Some test routing, some test routing, some routing with fees, some routing with fees and changes bias. 
We can definitively improve stuff, but lets do that in another PR:

**Pull request merged**